### PR TITLE
Add PUID & PGID support

### LINKED ISSUE #39: Add PUID & PGID support
`aria2` 有时候需要和`nfs` 之类的一起部署，会有权限问题，希望增加可以指定运行用户功能。
参考https://github.com/lukasmrtvy/lsiobase-aria2-webui

### PR DESCRIPTION
**Pull request by whlsxl**
fixed #39

### CODE REVIEW

**Comment by whlsxl** [comment]
haven't test `arm32` & `arm64`

File: Dockerfile [hunk 1]
```diff
@@ -1,35 +1,30 @@
-FROM alpine:edge
+FROM lsiobase/alpine:3.10
```
Comment (wahyd4) [inline Dockerfile]: I won't accept this base image

File: root/etc/cont-init.d/50-config [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/with-contenv bash
```
Comment (wahyd4) [inline root/etc/cont-init.d/50-config]: What's this used for?

File: arm32.Dockerfile [hunk 3]
```diff
@@ -1,41 +1,35 @@
...
+  && version=1.1.0 \
+  && wget -N --no-check-certificate https://github.com/mayswind/AriaNg/releases/download/${version}/AriaNg-${version}.zip && unzip AriaNg-${version}.zip && rm -rf AriaNg-${version}.zip \
+  && sed -i 's/6800/80/g' /usr/local/www/aria2/js/aria-ng*.js \
+  && apk del curl wget tar 
 
-RUN mkdir -p /usr/local/www && mkdir -p /usr/local/www/aria2
-ADD conf /app/conf
-COPY aria2c.sh caddy.sh Procfile /app/
-COPY Caddyfile SecureCaddyfile /usr/local/caddy/
+COPY root/ /
```
Comment (wahyd4) [inline arm32.Dockerfile]: where is the root folder comes from?

File: Dockerfile (hunk 1 above)
Comment (whlsxl) [reply Dockerfile]: This is from http://linuxserver.io , make it easy to use guid, completely open source

File: root/etc/cont-init.d/50-config (hunk 2 above)
Comment (whlsxl) [reply root/etc/cont-init.d/50-config]: This is `lsiobase/alpine`'s mechanism, every script will execute in order when boot

File: arm32.Dockerfile (hunk 3 above)
Comment (whlsxl) [reply arm32.Dockerfile]: Delete all files need to be copy to image, create `root/` by files location, make it easy to see the directory structure.

**Comment by whlsxl** [comment]
```
Removing intermediate container 2eeea3d70df9
 ---> 17ebc49928f4
Successfully built 17ebc49928f4
Successfully tagged wahyd4/aria2-ui:2.0.49
Password: 
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated
```
This is build log, the build is successfully, need a password for upload to docker repository?

**Comment by wahyd4** [comment]
No, only master will push docker push to remote

File: arm32.Dockerfile (hunk 3 above)
Comment (wahyd4) [reply arm32.Dockerfile]: So you mean the `root/` is the root folder of the application? 
Even though this is the reason, `root` it's still not a good name

File: Dockerfile (hunk 1 above)
Comment (wahyd4) [reply Dockerfile]: This base image makes the Dockerfile much harder to read, I guess my short answer is `NO`, but I'm happy to do some more some searching and spikes about this.

But anyway, huge thanks for your pull request. I can keep this open so you can try to simply this pitch as well.

**Comment by whlsxl** [comment]
> No, only master will push docker push to remote

but the build stuck on `password` after `successfully tagged` , do you have any idea what's happening?

File: Dockerfile (hunk 1 above)
Comment (whlsxl) [reply Dockerfile]: `linuxserver`  published almost every popular open source services in docker hub, most repositories already been downloaded over 10M+,  It's kind of "standard" to use `GUID`.

Comment (whlsxl) [reply Dockerfile]: Yes, It make `Dockerfile` a little harder to read, but compared to the features it provides, I think this is acceptable.

File: arm32.Dockerfile (hunk 3 above)
Comment (whlsxl) [reply arm32.Dockerfile]: It's more like a convention, many `linuxserver` images use `root/` as the image's root. I know It is easy to be confused with linux user root.

File: Dockerfile (hunk 1 above)
Comment (wahyd4) [reply Dockerfile]: I can see it's popular, but I don't it's the best way to every image, I did a draft pull request which did the same thing but from raw `alpine`, you can have a look.

For me `readability` and `maintenance` are the top two things can help me to keep improving this docker image.

**Comment by wahyd4** [comment]
@whlsxl I will close this pull request now, as I have another alternative solution with borrowing something from you. 

But anyway, thanks for contributing and hope you find this docker image useful.

**Pull request closed without merging**